pages.pt_BR/common/asdf.md

```diff
@@ -0,0 +1,28 @@
+# asdf
+
+> Interface de linha de comando para gestão de versões de diferentes pacotes.
```

Comment:
```suggestion
> Utilitário para a gestão de vários ambientes de execução de linguagens de programação
```

Reply:
@andrik não é apenas utilizado para linguagens de programação, deveria ser um termo mais genério

Reply:
@xRuiAlves Eu fiz o comentário baseado na descrição no [GH do projeto](https://github.com/asdf-vm/asdf)

> asdf-vm is a CLI tool that can manage multiple language runtime versions on a per-project basis.

Poderíamos refinar um pouco a descrição, pois particularmente achei ela muito genérica, podendo até gerar a interpretação que o asdf pode substituir um gerenciador de pacotes do próprio SO. O que você acha?

Reply:
Pois, estou um pouco na dúvida também. @schneiderl o que achas?

Reply:
@xRuiAlves e @schneiderl O que vocêss acham da seguinte descrição?

> Interface de linha de comando para gestão de versões de linguagens e programas.

Reply:
@xRuiAlves sou de acordo com a descrição do @andrik

Reply:
/cc @andrik, @schneiderl, and @xRuiAlves. I don't speak the language, so I'm not sure what's going on here.

What's the blocking factor on merging here?

Reply:
@sbrl I've marked the other conversations as resolved to make it clearer where the hold up is. The issue is that there's some ambiguity in how to describe this tool. It's not easy to translate technical terms without sounding too jargon-y (e.g. "runtimes" → "ambientes de execução"), whereas using more generic terms makes the description too vague (e.g. "packages"/"pacotes").

Looking at the [list of plugins](https://asdf-vm.com/#/plugins-all?id=plugin-list), I think we can all agree that it's hard to group them under a specific enough term that doesn't exclude parts of the list. They do seem to focus primarily on runtimes for interpreted languages, such as Node.js, Python and Ruby; but looking at the rest of the list, I can't think of anything better than "languages and programs" ("linguagens e programas") to properly encompass the at least most of the items there.

That said, it's redundant to have "de linha de comando" in the description, since CLI tools are exactly what tldr-pages covers anyway. So I'd go with @andrik's original phrasing for the first half of the sentence, and his latest proposal for the second half:

> Utilitário para a gestão de versões de linguagens e programas.

WDYT @andrik, @schneiderl, @xRuiAlves?

Reply:
@waldyrious I do agree that this is the best way we found to phrase this.
I think we can go with that.